pontoon/teams/static/js/bugzilla.js

```diff
@@ -99,4 +111,4 @@ var Pontoon = (function (my) {
 
     }
   });
-}(Pontoon || {}));
+}(Pontoon || {}));
```

Comment:
Please keep the newline at the end.

Reply:
Okay, thanks. Can you please explain this..."keep the newline at the end"

Reply:
You've removed an empty line at the end of file. Please add it back.

Here's why:
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file

Reply:
Okay, that makes sense.